database/migrations/sqlite/000001_create_blocks.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE blocks (
+                        inserted_at DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL,
+                        id INTEGER NOT NULL,
+                        hash BLOB NOT NULL,
+                        prevhash BLOB NOT NULL,
+                        merkleroot BLOB NOT NULL,
+                        height INTEGER NOT NULL,
+                        processed_at DATETIME,
+                        size INTEGER,
+                        tx_count INTEGER,
+                        orphanedyn BOOLEAN DEFAULT 0 NOT NULL,
+                        merkle_path TEXT DEFAULT ''
+);
+
+CREATE INDEX ix_blocks_hash ON blocks (hash);
```

Comment:
There is another index missing: https://github.com/bitcoin-sv/arc/blob/bbf2eab6ec8add99c2ffe0b3ff4aa7dc4096cae4/blocktx/store/sql/sql.go#L181
```
CREATE UNIQUE INDEX IF NOT EXISTS pux_blocks_height ON blocks(height) WHERE orphanedyn = FALSE;
```